package.json

```diff
@@ -16,6 +17,8 @@
     "@types/react-dom": "17.0.2"
   },
   "dependencies": {
```

Comment:
We should try to limit the amount of new packages

With this PR we have a increase in especially UI libraries which I think we should be able to reduce.
- emotion
- @mui
- @radix-ui
- tailwind
- _in addition what we already got some ui libraries_

In addition
- `@uidotdev/usehooks` I don't see used anywhere, could it be removed ? 
- `babelify` is used for solcworker2, but that isn't used anywhere, could it be removed ? 
    -  I believe the same is true for `path-browserify`
- I don't see `react-hook-form` used anywhere 
- I don't see `jsx-runtime` used anywhere

Reply:
usehooks, react-hook-form can be removed. idk about jsx-runtime.
path-browserify was important for solc, but I think I resolved it eventually. will try to remove.
babelify is used to babelify the babel directory.

Reply:
I removed everything I could and restored original version of packages I upgraded. I guess progress is a slow process.